frontend/src/metabase/timelines/selectors.ts

```diff
@@ -0,0 +1,18 @@
+import * as Urls from "metabase/lib/urls";
+import { State } from "metabase-types/store";
+import { ModalProps } from "./types";
+
+export const getTimelineId = (
+  state: State,
+  props: ModalProps,
+): number | undefined => {
+  return Urls.extractEntityId(props.params.timelineId);
+};
+
+export const getCollectionId = (state: State, props: ModalProps) => {
+  return Urls.extractCollectionId(props.params.slug);
+};
+
+export const getTimelineQuery = (state: State, props: ModalProps) => {
+  return { collectionId: getCollectionId(state, props) };
+};
```

Comment:
I believe people would usually expect functions from a `selectors` file to return something computed out of the redux state. Should we maybe rename the file or just use `Urls.extractEntityId` and `Urls.extractCollectionId` directly instead? It will be very explicit the IDs are extracted from a URL an that redux is not involved at all

Reply:
Agree. I think I initially took these from the store and forgot to change it. I've inlined all Urls calls and removed this file